gtamsolution.txt

```diff
@@ -0,0 +1,233 @@
+*/
+
+// Complete Exercise 8 below...
+var soyIdx= foods.indexOf('tofu')
+
+console.log('Exercise 8 Result:\n', soyIdx);
+
+/*
+Exercise 9:
+  - Assign to a variable named allFoods the result of joining the strings in the foods array such that the result is the following single string:
+    'taco -> sushi -> cupcake -> tofu -> cheeseburger'
+*/
+
+// Complete Exercise 9 below...
+var allFoods= foods.join(' -> ')
+
+console.log('Exercise 9 Result:\n', allFoods);
+
+/*
+Exercise 10:
+  - Assign a boolean to a variable named hasSoup depending upon whether or not the foods array includes the string 'soup'.
+*/
+
+// Complete Exercise 10 below...
+var hasSoup= foods.includes('soup') ;
+
+console.log('Exercise 10 Result:\n', hasSoup);
+
+
+/*
+Exercise 11:
+  - Use the forEach method to iterate through the provided nums array and add each odd number to a new array named odds.
+  - Hint: Initialize the odds variable to an empty array before the iteration.
+*/
+
+const nums = [100, 5, 23, 15, 21, 72, 9, 45, 66, 7, 81, 90];
+
+// Complete Exercise 11 below...
+ var odds= [];
+ nums.forEach(function(item, index){
+   if(item%2==1){
+     odds.push(item);
+   }
+
+ });
+
+console.log('Exercise 11 Result:\n', odds);
+
+/*
+Exercise 12:
+  - Use the forEach method to iterate through the same nums array and add the number to arrays named fizz, buzz and/or fizzbuzz based upon the following:
+  	- Add to the fizz array if the number is evenly divisible by 3.
+  	- Add to the buzz array if the number is evenly divisible by 5.
+  	- Add to the fizzbuzz array if the number is evenly divisible by 3 & 5.
+*/
+
+// Complete Exercise 12 below...
+var fizz= [];
+var buzz= [];
+var fizzbuzz= [];
+
+nums.forEach(addTo);
+
+function addTo(item){
+  if(item % 3 == 0 && item % 5 ==0){
+    fizzbuzz.push(item);
+    if(item% 3==0) {
+      fizz.push(item);
+      if(item%5 ==0){
+        buzz.push(item);
+      }
+    }
+    else if(item%5== 0){
+      buzz.push(item);
+    }
+} else if (item % 3 == 0){
+  fizz.push(item);
+} else if (item % 5 == 0){
+  buzz.push(item);
+}
+}
+
+console.log('Exercise 12 Results:');
+console.log('  fizz:', fizz);
+console.log('  buzz:', buzz);
+console.log('  fizzbuzz:', fizzbuzz);
+
+/*
+Exercise 13:
+  - Given the below numArrays array of arrays (two-dimensional array), assign the last nested array to a variable named numList.
+  - Assume you don't know how many nested arrays numArrays contains.
+*/
+
+const numArrays = [
+	[100, 5, 23],
+	[15, 21, 72, 9],
+	[45, 66],
+	[7, 81, 90]
+];
+
+// Complete Exercise 13 below...
+//console.log(numArrays.length);
+var numList= (numArrays.slice(numArrays.length-1,numArrays.length));
+
+console.log('Exercise 13 Result:\n', numList);
+
+/*
+Exercise 14:
+  - Given the above numArrays array, access the number 66 and assign to a variable named num.
+*/
+
+// Complete Exercise 14 below...
+
+var num= numArrays[2][1];
```

Comment:
nice! can you make this more programmatic (so that it will work no matter where the number `66` is)? Hint: `indexOf()`